src/sinks/new_relic_logs.rs

```diff
@@ -249,7 +255,7 @@ mod tests {
         region = "eu"
 
         [batch]
-        max_size = 8388608
+        max_size = 838860
```

Comment:
I'm changing the max_size to be under max limit here, so that the test should pass, and adding another test which should panic below.